src/utils.ts

```diff
@@ -8,6 +8,13 @@ export function getOrKey(key: string, obj: { [s: string]: string }): string {
 export const delay = (time: number) =>
   new Promise(resolve => setTimeout(resolve, time));
 
+export function fixCredit(minUnits: number, maxUnits: number) : string {
+  if(minUnits > 0) {
```

Comment:
Gonna merge this anyways but please use [prettier](https://prettier.io/) for formatting